README.rst

```diff
@@ -0,0 +1,67 @@
+==============================
+REANA Environment VOMSproxy
+==============================
+
+About
+=====
+
+``reana-auth-vomsproxy`` provides a container image with encapsulated runtime
+execution environment for creating a VOMS proxy certificate. The container image
+includes no additional logic or libraries, just the bare minimum to support the VOMS client. 
+The image is configured to support authentication for the four experiments at CERN; ALICE, ATLAS, CMS and LHCb.
+
+``reana-auth-vomsproxy`` was developed for use in the `REANA <http://reana.readthedocs.io/>`_ reusable research data analysis platform.
+
+Usage
+=====
+
+You can use ``reana-auth-vomsproxy`` as a base image, however it was built
+to be used as a sidecar container with the single purpose of refreshing the proxy certificate. 
+Once obtained, the certificate is shared with the main container using common namespaces. 
+
+Some concrete usage examples will be provided later on.
+
+Development
+===========
+
+You can build the ``reana-auth-vomsproxy`` image by passing the environment variables ``DATE`` in
+the format "YYYY-MM-DD" and ``VERSION`` as either a semantic versioning and/or the git SHA.
+
+Running the container and successfully obtaining a VOMS proxy certificate requires additional information and inputs:
+
+- Grid user certificate: ``${HOME}/.globus/usercert.pem``
+- Grid user key: ``${HOME}/.globus/userkey.pem``
+- Grid user password
+- Virtual organisation membership (e.g. for CMS: ``cms``)
+
+If you would like to try it locally, you can run:
+
+.. code-block:: console
+
+   $ docker run -i -t --rm -v $HOME/foo:/root/.globus/ reana-auth-vomsproxy /bin/bash
+
+
+Your local directory ``/foo`` should contain your ``usertcert.pem`` and ``userkey.pem`` files. By default the VOMS client
+checks the directory ``/$HOME/.globus`` for the files needed. In this image that path results in ``/root/.globus/``.
+
+A proxy can then be obtained by specifying the Virtual Organization, for example via:
+
+.. code-block:: console
+
+  $ voms-proxy-init --voms cms
```

Comment:
I would place all of these instructions under _Usage_, because it is actually how you configure and how to run it, right?

Development can be left to talk just about building the image, as in https://github.com/reanahub/reana-env-jupyter#development. This reminds me that we are not having a Makefile as in other `reana-env-_`s, I am working on the `reana-env-krb5`, I will add it to my PR as well so the development instructions are the same everywhere, what do you think about adding it here?

Reply:
Yes you're right, it fits better under _Usage_. I'll move it! 

Having a Makefile sounds good, I'll take example from your krb5.